translations/locale_fr.json

```diff
@@ -0,0 +1,535 @@
+				},
+				"example": {
+					"title": "Exemple",
+					"intro": "Voyons un exemple de jeu. Nous allons d'abord essayer une créature arbitraire.",
+					"table_interpretation_1": "À partir de cet essai, nous pouvons voir que la créature que nous cherchons a été ajoutée plus récemment que la version 1.6.1, qu'elle n'a pas beaucoup de vie, qu'elle mesure moins de 1,6 blocs et qu'elle peut être passive ou hostile, ou les deux à la fois. Elle n'est pas invoquée par la foudre et n'est pas morte-vivante non plus.",
+					"table_interpretation_2": "D'accord, c'est encore plus récent que la version 1.13, mais nous avons la vie : nous cherchons une créature qui a 10 de vie, c'est-à-dire 5 cœurs. Sa taile est de moins de 0,6 blocs, donc nous cherchons une petite créature. Son comportement est encore une fois une correspondance partielle, donc elle est très probablement passive, mais ça pourrait encore être hostile et neutre. L'apparition et la classification correspondent partiellement, donc apparition \"Biome\" et quelque chose d'autre. Elle est classée comme Animal ou Aquatic, mais peut-être quelque chose d'autre.",
+					"table_interpretation_3": "Super ! Regarde qui a sauté dans ce tutoriel. Nous avons trouvé une petite Grenouille !"
+				},
+				"partial_matches": {
+					"title": "Explication de la Correspondance Partielle",
+					"intro": "Examinons en détail les correspondances partielles, car c'est probablement la couleur de case qui prête le plus à confusion.",
+					"explanation": "Une correspondance partielle ne peut apparaître que dans les champs constitués d'une liste (par exemple Apparition, Classification ou Comportement). Si tu as choisi une créature avec au moins un élément de liste qui correspond à la créature à trouver, tu obtiendras une correspondance partielle. Voici un exemple pour les chevaux :"
+				},
+				"tips": {
+					"title": "Astuces",
+					"not_all_blocks": "MCdle ne contient pas tous les types d'entités. Seules les entités vivantes, comme par exemple les cochons ou les zombies, sont incluses. Les porte-armures, les projectiles ou autres entités similaires non vivantes ne sont pas incluses. La liste est tirée du Minecraft Wiki, donc si une créature ne figure pas dans la liste, elle n'est pas présente dans le jeu.",
+					"utilize_the_search": "Utilise bien la barre de recherche ! En regardant les suggestions, tu pourras affiner ta recherche.",
+					"use_the_wiki": "N'hésite pas à utiliser le Minecraft wiki. Dès que tu as déterminé la version d'ajout du bloc, tu peux utiliser le wiki pour trouver les blocs qui ont été ajoutés dans cette version, ce qui t'aidera à limiter drastiquement tes recherches.",
+					"health_measuring": "Garde à l'esprit que la Vie est mesurée en points de santé, alias demi-cœurs, donc 10 points de vie correspondent à 5 cœurs."
```

Comment:
The minecraft font I use doesn't support this `œ` character. It ain't really a problem, just wanted to mention it. I could look for a different font or maybe there is some escape strategy. E.g. in german we got `ä`, `ö`, `ü`, which can all be escaped with `ae`, `oe` and `ue`. Curious if there is something like that in french. But as I said, not really an issue and doesn't need to be fixed essentially.
![image](https://github.com/ElitoGame/mcdle-public/assets/43675593/d3efb9fa-7c0f-4b37-845b-3188e37ef35d)